### PR TITLE
Stabilize NEP-488 (Host Functions for BLS12-381 Curve Operations)

### PR DESCRIPTION
**Pull request by olga24912**
Stabilize NEP-488: Host Functions for BLS12-381 Curve Operations

* NEP-488: https://github.com/near/NEPs/pull/488
* Implementation: https://github.com/near/nearcore/pull/9317

### CODE REVIEW

File: core/parameters/src/config_store.rs [hunk 1]
```diff
@@ -45,8 +45,8 @@ static CONFIG_DIFFS: &[(ProtocolVersion, &str)] = &[
     (69, include_config!("69.yaml")),
     // Introduce ETH-implicit accounts.
     (70, include_config!("70.yaml")),
+    (71, include_config!("71.yaml")),
```
Comment (nagisa) [inline core/parameters/src/config_store.rs:48]: May or may not want to re-use protocol version 70, as the next mainnet release will be going up to 69. And 70 already comes in the release after the upcoming one. Doesn't matter much though, AFAIU protocol is now able to upgrade multiple protocol versions per release.

**Review by nagisa — APPROVED** [review]
LGTM as far as the stabilization itself is concerned.

File: chain/jsonrpc/jsonrpc-tests/res/genesis_config.json [hunk 2]
```diff
@@ -1,5 +1,5 @@
 {
-  "protocol_version": 70,
+  "protocol_version": 71,
```
Comment (staffik) [inline chain/jsonrpc/jsonrpc-tests/res/genesis_config.json:2]: We want to release BLS signatures as part of protocol version 70 release.

File: core/primitives-core/src/version.rs [hunk 3]
```diff
@@ -211,6 +210,7 @@ impl ProtocolFeature {
             | ProtocolFeature::RemoveAccountWithLongStorageKey => 68,
             ProtocolFeature::StatelessValidation => 69,
             ProtocolFeature::EthImplicitAccounts => 70,
+            ProtocolFeature::BLS12381 => 71,
```
Comment (staffik) [inline core/primitives-core/src/version.rs:213]: We want `BLS12381` feature to be released together with `StatelessValidation`. Similarly as protocol v69 contains several features.

File: chain/jsonrpc/jsonrpc-tests/res/genesis_config.json (hunk 2 above)
Comment (olga24912) [reply chain/jsonrpc/jsonrpc-tests/res/genesis_config.json]: fixed: https://github.com/near/nearcore/pull/11818/commits/ac29a2646c49bcdfbee9b8c335191e007a2e2ad3

https://github.com/near/nearcore/pull/11818/commits/2e39777b34acf4cf1b97a52b2b8c6ce292050b27

File: core/primitives-core/src/version.rs (hunk 3 above)
Comment (olga24912) [reply core/primitives-core/src/version.rs]: fixed: https://github.com/near/nearcore/pull/11818/commits/ea0400f21f86194ddcd9ac630335c7abb59cbb46

File: core/parameters/src/config_store.rs (hunk 1 above)
Comment (olga24912) [reply core/parameters/src/config_store.rs]: fixed: https://github.com/near/nearcore/pull/11818/commits/ea0400f21f86194ddcd9ac630335c7abb59cbb46

**Pull request merged**